UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/Services/BuilderService.cs

```diff
@@ -46,11 +52,19 @@ public async Task<string> CreateBespokeExchangeSetAsync(ConfigQueueMessage confi
 
             ExtractExchangeSetZip(essFiles, essFileDownloadPath);
 
+            var exchangeSetPath = Path.Combine(homeDirectoryPath, essBatchId, fssApiConfig.Value.BespokeExchangeSetFileFolder);
+
+            await PerformAncillaryFilesOperationsAsync(essBatchId, exchangeSetPath, configQueueMessage.CorrelationId, configQueueMessage.ReadMeSearchFilter);
```

Comment:
```suggestion
if (configQueueMessage.ReadMeSearchFilter != ReadMeSearchFilter.AVCS.ToString())
            await PerformAncillaryFilesOperationsAsync(essBatchId, exchangeSetPath, configQueueMessage.CorrelationId, configQueueMessage.ReadMeSearchFilter);
```

Reply:
Applied changes as suggested by Nadeem as we are performing ancillary files operations into this method not only readme file related operations.